Tests/UnitTests/Core/Vertexing/AdaptiveMultiVertexFitterTests.cpp

```diff
@@ -152,15 +154,11 @@ BOOST_AUTO_TEST_CASE(adaptive_multi_vertex_fitter_test) {
   }
 
   std::vector<Vertex<BoundTrackParameters>*> vtxPtrList;
+  ACTS_DEBUG("All vertices in test case:");
   int cv = 0;
-  if (debugMode) {
-    std::cout << "All vertices in test case: " << std::endl;
-  }
   for (auto& vtx : vtxList) {
-    if (debugMode) {
-      cv++;
-      std::cout << "\t" << cv << ". vertex ptr: " << &vtx << std::endl;
-    }
+    cv++;
+    ACTS_DEBUG("\t" << cv << ". vertex ptr: " << &vtx);
```

Comment:
```suggestion
    ACTS_DEBUG("\t" << &vtx - vtxList.begin() + 1 << ". vertex ptr: " << &vtx);
```
You could try this approach, if you want to completely get rid of the debug-variable `cv`.

Could also work for some loops below.